### PR TITLE
Add Repeat string extension

### PR DESCRIPTION
**Pull request by kasuken**

### CODE REVIEW

**Comment by kasuken** [comment]
> for (int i = 0; i < count; i++)
> 
> non deve essere for (int i = 0; i < times; i++) ?

I think you are seeing an old commit.
In the last commit it's solved that issue.

File: src/TinyHelpers/Extensions/StringExtensions.cs [hunk 1]
```diff
@@ -21,4 +22,14 @@ public static string ReplaceIgnoreCase(this string input, string pattern, string
 
     public static string? GetValueOrDefault(this string? input, string? defaultValue = default, bool whitespaceAsEmpty = true)
         => whitespaceAsEmpty ? (string.IsNullOrWhiteSpace(input) ? defaultValue : input) : (string.IsNullOrEmpty(input) ? defaultValue : input);
+
+    public static string Repeat(this string str, int times)
```
Comment (marcominerva) [inline src/TinyHelpers/Extensions/StringExtensions.cs:26]: Use meaningful variable and parameter names. Replace `str `with `input `(like in the other methods of this class).

File: src/TinyHelpers/Extensions/StringExtensions.cs [hunk 2]
```diff
@@ -21,4 +22,14 @@ public static string ReplaceIgnoreCase(this string input, string pattern, string
...
+    {
+        var s = new StringBuilder();
```
Comment (marcominerva) [inline src/TinyHelpers/Extensions/StringExtensions.cs:28]: Use a meaningful variable name. Replace `s` with something like `result` or `output`.

File: src/TinyHelpers/Extensions/StringExtensions.cs [hunk 3]
```diff
@@ -21,4 +22,14 @@ public static string ReplaceIgnoreCase(this string input, string pattern, string
...
+        for (int i = 0; i < times; i++)
```
Comment (marcominerva) [inline src/TinyHelpers/Extensions/StringExtensions.cs:29]: replace `int` with `var` (suggestion: use Code Clean Up to automatically follow the guidelines).

File: src/TinyHelpers/Extensions/StringExtensions.cs [hunk 4]
```diff
@@ -21,4 +22,14 @@ public static string ReplaceIgnoreCase(this string input, string pattern, string
...
+        {
+            s.Append(str);
+        }
+        return s.ToString();
```
Comment (marcominerva) [inline src/TinyHelpers/Extensions/StringExtensions.cs:33]: Missing blank line after curly brace.

File: src/TinyHelpers/Extensions/StringExtensions.cs [hunk 5]
```diff
@@ -21,4 +22,14 @@ public static string ReplaceIgnoreCase(this string input, string pattern, string
 
     public static string? GetValueOrDefault(this string? input, string? defaultValue = default, bool whitespaceAsEmpty = true)
         => whitespaceAsEmpty ? (string.IsNullOrWhiteSpace(input) ? defaultValue : input) : (string.IsNullOrEmpty(input) ? defaultValue : input);
+
+    public static string Repeat(this string str, int times)
+    {
+        var s = new StringBuilder();
+        for (int i = 0; i < times; i++)
+        {
+            s.Append(str);
```
Comment (marcominerva) [inline src/TinyHelpers/Extensions/StringExtensions.cs:31]: What happens if `str` is null or `times` is negative? Perhaps it makes sense to add some parameter checks above.

**Review by marcominerva — CHANGES_REQUESTED** [review]
Please take a look to the comments. Thank you :-)

**Comment by marcominerva** [comment]
Closed as no reply from the author

**Pull request closed without merging**